src/engine/sync/enginesync.cpp

```diff
@@ -498,10 +502,12 @@ void EngineSync::addSyncableDeck(Syncable* pSyncable) {
 
 void EngineSync::onCallbackStart(int sampleRate, int bufferSize) {
     m_pInternalClock->onCallbackStart(sampleRate, bufferSize);
+    m_pAbletonLink->onCallbackStart(sampleRate, bufferSize);
 }
 
 void EngineSync::onCallbackEnd(int sampleRate, int bufferSize) {
     m_pInternalClock->onCallbackEnd(sampleRate, bufferSize);
+    m_pAbletonLink->onCallbackStart(sampleRate, bufferSize);
```

Comment:
Is the time advanced a second time here?

Reply:
yes this looks like a typo, should be onCallbackEnd